src/settings/project/mod.rs

```diff
@@ -143,144 +139,116 @@ impl Manifest {
                             false // workers_dot_dev defaults to false when it's top level and --release is passed
```

Comment:
Does it actually make sense to bail! on the Some(_) case above, given that the user is actually using `workers_dot_dev`? I would expect for us to just follow what `workers_dot_dev` specifies in this case. This surprised me a little--let me know if I'm thinking incorrectly!

Also, in the None case where we return false, should we modify the warning message to also clarify that we are falling back to deploying to a route?

Reply:
We don't want `--release` and `workers_dot_dev` to be compatible, if they are using `workers_dot_dev`, they have opted in to the new type of `wrangler.toml` and should not be using the `--release` flag

Reply:
Gotcha :)